content/numpy/concepts/built-in-functions/terms/dot/dot.md

```diff
@@ -0,0 +1,84 @@
+---
+Title: '.dot()'
+
+Description: 'Dot product of two arrays.' 
+
+Subjects: 
+  - 'Data Science'
+  - 'Computer Science'
+
+Tags: 
+  - 'Data Structures'
+  - 'Arrays'
+  - 'Linear Algebra'
+  - 'Functions'
+  - 'NumPy'
+
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'  
+---
```

Comment:
```suggestion
---
Title: '.dot()'
Description: 'Computes the dot product of two arrays.' 
Subjects: 
  - 'Computer Science'
  - 'Data Science'
Tags: 
  - 'Arrays'
  - 'Data Structures'
  - 'Functions'
  - 'Linear Algebra'
  - 'NumPy'
CatalogContent: 
  - 'learn-python-3'
  - 'paths/computer-science'
  - 'paths/data-science'
  - 'paths/data-science-foundations'  
---
```

Reply:
Please accept the above suggestion; there should not be any line spacing in the meta data and it should be alphabetically arranged.